treenode.go

```diff
@@ -126,9 +128,9 @@ func (n *TreeNodeInstance) SendTo(to *TreeNode, msg interface{}) error {
 	var c *GenericConfig
 	// only sends the config once
 	n.configMut.Lock()
-	if n.config != nil {
+	if !n.sentTo[to.ID] {
 		c = n.config
-		n.config = nil
+		n.sentTo[to.ID] = true
 	}
```

Comment:
Should we add a `n.config = nil` once the configuration has been sent to all nodes? This would help the garbage-collector.

On the other hand, the protocol is supposed to finish _some_ time, then it will be cleaned up a bit later.